app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ Required ] According to the project requirements when a user creates an account the application should ask the user to confirm the email address they used to sign up. Please consider adding the `confirmable` attribute of `devise` to your `user` model to enable this.

Reply:
Issue has been solved.